caas/kubernetes/provider/k8s.go

```diff
@@ -1118,9 +1118,6 @@ func (k *kubernetesClient) EnsureService(
 	if params == nil || params.PodSpec == nil {
 		return errors.Errorf("missing pod spec")
 	}
-	if params.PodSpec.OmitServiceFrontend && len(params.Filesystems) == 0 {
-		return errors.Errorf("kubernetes service is required when using storage")
```

Comment:
what's the initial purpose to do this check here?

Reply:
not sure anymore